bundles/GlossaryBundle/public/js/glossary.js

```diff
@@ -11,8 +11,8 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.settings.glossary");
-pimcore.settings.glossary = Class.create({
+pimcore.registerNS("pimcore.bundle.glossary");
+pimcore.bundle.glossary = Class.create({
```

Comment:
```suggestion
pimcore.registerNS("pimcore.bundle.glossary.settings");
pimcore.bundle.glossary.settings = Class.create({
```

Reply:
I renamed the namespaces and also changed the glossary.js to settings.js to make it more consistent with the namespace. 
Did the same with the startup.js so the namespace is pimcore.bundle.glossary.startup to be consistent with the filenames.

Reply:
👍 LGTM